locales/en.json

```diff
@@ -139,14 +139,16 @@
       "go-to-contributors": "Go to contributors",
       "more-info": "More information",
       "show-all": "Show all",
-      "remove-from-playlist": "Remove from playlist"
+      "remove-from-playlist": "Remove from playlist",
+      "transcription": "Transcription"
```

Comment:
Since you're norwegian: can you add the norwegian translation in Crowdin once this PR is merged?